grpc.go

```diff
@@ -515,6 +515,34 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	for idx, ociMnt := range ociSpec.Mounts {
+		splitSourceSlice := strings.Split(ociMnt.Source, "/")
+		if len(splitSourceSlice) > 1 {
+			k8sStorageType := splitSourceSlice[len(splitSourceSlice)-2]
```

Comment:
You could create a variable for `len(splitSourceSlice)` as you've scanned the string twice here.

Reply:
sure.